tests/jeos/firstrun.pm

```diff
@@ -70,8 +70,16 @@ sub run {
 
     # JeOS on generalhw
     mouse_hide;
+    # expected in near future if (is_tumbleweed || is_leap('15.4+') || is_sle('15-sp4+') {
+    unless (is_leap('15.4+') || is_sle('15-sp4+')) {
+        assert_screen 'jeos-init-config-screen', 300;
+        wait_still_screen;
```

Comment:
Needs a comment why this is necessary (if it even still is)

Reply:
I think that would require a lot of testing to avoid stability issues. The situation is similar to condition on L#86 and its body, as both are handling initial screens I have rather kept it there.